### ISSUE TITLE
amp-date-picker i18n and customization for a11y phrases

### ISSUE THREAD

**Issue by cvializ** [ampproject/amphtml#13747]
The `amp-date-picker` experimental MVP supports a11y phrases that are read by screen readers, as seen in `defaultPhrases.js`. However the MVP lacks support for customizing these phrases for publisher-specific wording or i18n. This issue will allow publishers to specify their own phrases in the HTML or JSON API for `amp-date-picker`.

**Comment by aghassemi** [comment]
:+1:  I vote for an optional `<script>JSON<script>` child (and no remote option)

**Comment by ampprojectbot** [comment]
This is a high priority issue but it hasn't been updated in awhile. @cvializ Do you have any updates?

**Comment by 88kbbq** [comment]
I just tried to implement amp-date-picker and found that the i info button does not respond to the "locale" attribute. We have a multi-lingual site. This is really important to be able to change the locale. How can I customize the months and info section to Spanish, Chinese and Japanese?

**Comment by cvializ** [comment]
At the moment internationalization for the info button is not implemented. The months are customizable [using the `locale` attribute](https://www.ampproject.org/docs/reference/components/amp-date-picker#locale). Does the `locale` attribute work for changing the month and days? e.g. `locale=zh-CN`, `locale="es"`, `locale="ja"`

**Comment by ampprojectbot** [comment]
This is a high priority issue but it hasn't been updated in awhile. @cvializ Do you have any updates?

**Comment by ampprojectbot** [comment]
This issue hasn't been  updated in awhile. @cvializ Do you have any updates?
